src/app/app.component.css

```diff
@@ -5,11 +5,12 @@
     /* Set the fixed height of the footer here */
     height: 60px;
     line-height: 60px; /* Vertically center the text there */
-    background-color: #343434da;
+    background-color: #343434;
```

Comment:
haha I am wondering if "da" really was one of the issues.  That alpha value was based upon the level of the color's transparency. @kstarrmitche

Reply:
At this point I can just say that I am glad it works for everyone.

Reply:
Unfortunately HEXA is relatively new and not all browsers support it. I assume there's a way to code in a fallback in css in case of issues like this (css filter?). A good alternative to the 8 digit hex value if you really care about transparency is by using RGBA instead, which has wider support.

Reply:
I was just messing with it's transparency through the built in "color-picker" within VSC. I didn't know it would present a problem. But this is good for future reference.

Reply:
np :) The hardest part about coding for the web is browser compatibility IMO - it's hard to test for that!!